### PR TITLE
[CoSim] imports cleaning

### PR DESCRIPTION
**Pull request by philbucher**
This PR cleans/simplifies a bit the py-imports, same as I have done in other apps

### CODE REVIEW

**Comment by adityaghantasala** [comment]
Why relative imports ?

**Comment by philbucher** [comment]
why not?
Its shorter and to me more clear

**Comment by adityaghantasala** [comment]
its shorter yes ... 
but is not convenient and clear for reading .. one should always track the location of the file .. !
I clearly agree with this guy here 
https://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module

**Comment by philbucher** [comment]
hm ok, then I will change it back

**Comment by philbucher** [comment]
@adityaghantasala done

**Review by adityaghantasala — APPROVED** [review]

**Comment by philbucher** [comment]
thx

File: applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_coupled_solver.py [hunk 1]
```diff
@@ -245,13 +250,10 @@ def Check(self):
             coupling_operation.Check()
 
     def __CreateSolverWrappers(self):
-        ### ATTENTION, big flaw, also the participants can be coupled solvers !!!
```
Comment (philbucher) [inline applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_coupled_solver.py:248]: @adityaghantasala btw this comment is outdated, using coupled solvers also works now

**Pull request merged**